.github/workflows/deploy-to-gcp.yaml

```diff
@@ -37,8 +41,32 @@ jobs:
           sudo snap install task --classic
 
       - name: Terraform Plan
+        if: github.event_name == 'pull_request'
```

Comment:
Do we need to explicitly call out the event_name to be `pull_request` here given we already define this on top of the file?
```
on: 
  push:
  pull_request:
```

Reply:
Applied